Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/DecreasesTo2.dfy.expect

```diff
@@ -0,0 +1,5 @@
+DecreasesTo2.dfy(5,24): Error: invalid UnaryExpression
+DecreasesTo2.dfy(9,21): Error: ident expected
+DecreasesTo2.dfy(9,12): Error: expected 'to'
+DecreasesTo2.dfy(13,10): Error: invalid ParensExpression
```

Comment:
I would love to see more helpful error messages, but they're just as good as the ones for other operators, so that's what we probably have to settle for.

Reply:
Yeah, I agree, on both sides.